bitnami/ghost/README.md

```diff
@@ -384,6 +384,11 @@ Find more information about how to deal with common errors related to Bitnami's
 
 ## Upgrading
 
+
+### To 19.0.0
+
+This major release changes database management system from MariaDB to MySQL. No major issues are expected during the upgrade.
```

Comment:
As this new major introduces a new Database, users coming from a previous version might be interesting in migrating their data and importing it back into the new database. There is an official guide on doing so, I would mention it in the upgrade notes as well 😁 

https://ghost.org/docs/migration/ghost/

WDYT?